### PR TITLE
Support for pre-release of OpenAFS + newer ansible versions

### PR DESCRIPTION
**Pull request by rbrunckhorst**

### CODE REVIEW

File: plugins/modules/openafs_principal.py [hunk 1]
```diff
@@ -689,6 +689,12 @@ class OracleMITKerberosAdmin(MITKerberosAdmin):
     kadm5_acl = '/var/kerberos/krb5kdc/kadm5.acl'
 
 
+class OracleMITKerberosAdmin(MITKerberosAdmin):
```
Comment (meffie) [inline plugins/modules/openafs_principal.py:692]: I think the class name should be opensuseMITKerberosAdmin.  (copy and paste error?)

**Review by meffie — COMMENTED** [review]
lgtm

**Review by meffie — APPROVED** [review]
Nice. Thanks.

File: plugins/modules/openafs_principal.py (hunk 1 above)
Comment (meffie) [reply plugins/modules/openafs_principal.py:692]: I'll merge the pr, then fix the class name.

**Comment by rbrunckhorst** [comment]
Upps yes. I have overseen that.

Regards,
-- 
Ralf Brunckhorst
***@***.***

**Comment by meffie** [comment]
I merged this, and fixed the class name.
Now I see why we had the "warn: false' attributes. Without it the ansible-lint issues an error, but the "warn:" attribute was deprecated in ansible 2.11 and then removed recently. Any time we run 'rpm' as a command, ansible-lint fails. :(

**Comment by meffie** [comment]
I added the new error to the ansible-lint skip list.

**Pull request merged**